ui/ui-frontend/projects/collect/src/app/collect/archive-search-collect/archive-preview/archive-preview.component.ts

```diff
@@ -73,23 +74,37 @@ export class ArchivePreviewComponent implements OnInit, OnChanges {
   ngOnInit() {}
 
   emitClose() {
-    this.isPanelextended = false;
     this.previewClose.emit();
+    this.isPanelextended = false;
     this.backToNormalLateralPanel.emit();
     this.selectedIndex = 0;
   }
 
+  selectedTabChangeEvent($event: MatTabChangeEvent) {
```

Comment:
if($event.index===0){ 

this.isPanelextended = false; this.backToNormalLateralPanel.emit(); }
  
  else if($event.index===1 || $event.index===2){

 this.isPanelextended = true; this.showExtendedLateralPanel.emit();
 
  } OU
  
   switch ($event.index) {
      case 0:
        this.isPanelextended = false;
        this.backToNormalLateralPanel.emit();
        break;
      case 1:
      case 2:
        this.isPanelextended = true;
        this.showExtendedLateralPanel.emit();
        break;
 
    }
    this.selectedIndex = $event.index;
  }